eng/Packages.Data.props

```diff
@@ -90,7 +90,7 @@
     <PackageReference Update="Azure.Storage.Blobs" Version="12.8.0" />
     
     <!-- Other approved packages -->
-    <PackageReference Update="Microsoft.Azure.Amqp" Version="2.4.13" />
+    <PackageReference Update="Microsoft.Azure.Amqp" Version="2.5.1" />
```

Comment:
>2.5.1 [](http://example.com/codeflow?start=61&length=5)

Why not 2.4.14? This is minor change only.

Reply:
The ability to use `Timeout.Zero` was added in 2.5.1.    Before that, the AMQP library would see the zero value and assume that we wanted  to apply the default.